.github/workflows/sca.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         run: |
           ./dev-tools/check_file_permissions.sh
           ./dev-tools/check_trailing_spaces.sh
-          ./dev-tools/vendor/bin/phpstan analyse
+          ./dev-tools/vendor/bin/phpstan analyse --no-progress
```

Comment:
why?
we do progress for other tools, eg for PHP CS Fixer itself

Reply:
To not have this "beautiful" logs:
![image](https://user-images.githubusercontent.com/9282069/102410572-05afaf00-3ff1-11eb-9ac8-fb71c05ff10c.png)
AFAIK other tools [don't](https://pipelines.actions.githubusercontent.com/6HzPVZnYJGwfOotYxu8ZJJIcqdtpCW1gc7jXnYZ3NyGFf1CXiD/_apis/pipelines/1/runs/857/signedlogcontent/3?urlExpires=2020-12-16T21%3A51%3A21.1008647Z&urlSigningMethod=HMACV1&urlSignature=MsjzJk%2FEv2KQE80npXNeKmj7CpAw6s4QyXn1XUBkaeI%3D) have such progress.

Reply:
> Uri expired

composer checker doesn't have it
phpmd doesn't have it, still (we are longly awaiting for it, the tool looks to froze)
php-cs-fixer we run with `-v` for progress

i would keep the progress indicator to show sth is happening for non-instant tools, eg phpstan

what's the downside of having it? if it's green, it's getting collapsed.

Reply:
> what's the downside of having it?

I was asking myself the opposite question: what is the benefit of having it? And could not find any.

Reply:
i know it's not frozen and progressing.
i was struggling with it a lot for slow tool like phpmd called for whole project at once, so i had to workaround it with checking only modified files. for phpstan we are checking whole codebase